model/app.model.ts

```diff
@@ -0,0 +1,84 @@
+export type NavLink = {
+  name: string
+  href: string
+}
+
+export type CatImage = {
+  id: string
+  url: string
+  width: number
+  height: number
+  breeds?: CatBreed[]
+  favourite?: {}
+}
+
+/*

```

Comment:
I guess these comments can be safely deleted?

Reply:
resolved